README.md

```diff
@@ -71,9 +71,58 @@ Seats, K. J., Jesse F. L., and German A. P. "Improved ambient noise correlation
 *Jiang, C. and Denolle, M. "NoisePy: a new high-performance python tool for seismic ambient noise seismology." _Seismological Research Letter_ 91, no. 3 (2020): 1853–1866..
 ** Yuan, C., Bryan, J. T., and Denolle, M. "Numerical comparison of time-, frequency- and wavelet-domain methods for coda wave interferometry." _Geophysical Journal International_ 226, no. 2 (2021): 828-846.
 
+## Parameters to configure
```

Comment:
@mdenolle I'd prefer not to duplicate this here ([DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)), since it could be error prone if the code and the README diverge. An alternative is to put a link here to `blob/master/src/noisepy/seis/datatypes.py`. This will point people to the source of truth for the parameters and the default values. We can even move the `ConfigParameters` class to its own file so that it's easier to link directly to it.

Reply:
Interesting. I think the configuration parameters are the most important information for the users and should be highly visible somewhere. You are right that a readme is probably not appropriate, should we consider in the docs/examples/tutorial.rst or docs/examples/example.rst?

Reply:
Yeah, I think the docs are a better place for this. I see there's already some code in the repo for building them with sphinx. I think the best path is to connect this to readthedocs.io so that the docs are built and published on every PR. This will generate a nice documentation page right from the source code, hence removing the need for duplication. It would look something like this: https://example-python.readthedocs.io/en/latest/example.html.

I'm ok with placing the parameters temporarily somewhere else and then moving to readthedocs later on...